lesson06/assignment/grid.html

```diff
@@ -0,0 +1,75 @@
+        <link rel="stylesheet" href="css/style.css" />
+        <title>Assignment 06 - Bootstrap 4</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+    </head>
+    <body>
+        <!--Header with responsive image-->
+        <header>
+          <img class="img-fluid" src="css/images/header2.JPG" alt="red tulips surrounded by purple tulips">
+        </header>
+        <!--Responsive Bootstrap navbar-->
+        <nav class="navbar navbar-expand-lg navbar-light m-3">
+            <a class="navbar-brand text-danger" href="#"><strong>Tulip Fever</strong></a>
+            <div class="navbar-nav">
+              <a class="nav-item nav-link text-danger" href="index.html">Home</a>
+              <a class="nav-item nav-link text-danger" href="images.html">Images</a>
+              <a class="nav-item nav-link text-danger active" href="#">Grid<span class="sr-only">(current)</span></a>
+              <a class="nav-item nav-link text-danger" href="accordion.html">Accordion</a>
+            </div>
+        </nav>
+
+        <div class="container">
+            <p>Visit us year-round to see what new and rare varieties are available.</p>
+        </div>
+
+        <!--Boostrap grid responsive to xs, small, medium, large, xl sizes-->
+        <div class="container">
+            <div class="row">
+              <div class="d-flex col-xs-8 col-sm-11 col-md-11 col-lg-12 col-xl-12 p-5" style="background-color: #c7deec;">
+                <div class="flex-fill m-2"><strong>Spring:</strong></div>
+                <div class="flex-fill m-2">March 1-May 31</div>
+                <div class="flex-fill m-2">Hours: Everyday, 10 a.m.-7 p.m.</div>
+              </div>
+
+              <div class="d-flex col-xs-8 col-sm-11 col-md-11 col-lg-12 col-xl-12 p-5" style="background-color: white;">
+                <div class="flex-fill m-2"><strong>Summer:</strong></div>
+                <div class="flex-fill m-2">June 1-Aug. 31</div>
+                <div class="flex-fill m-2">Hours: Everyday, 8 a.m.-9 p.m.</div>              
+              </div>
+
+              <div class="d-flex col-xs-8 col-sm-11 col-md-11 col-lg-12 col-xl-12 p-5" style="background-color: #c7deec;">
+                <div class="flex-fill m-2"><strong>Fall:</strong></div>
+                <div class="flex-fill m-2">Sep. 1-Nov. 30</div>
+                <div class="flex-fill m-2">Hours: W-Sun, 10 a.m.-7 p.m. <br>(Closed Monday & Tuesday)</div>            
+              </div>
+
+              <div class="d-flex col-xs-8 col-sm-11 col-md-11 col-lg-12 col-xl-12 p-5" style="background-color: white;">
```

Comment:
As commented above, there should never be embedded styles in HTML elements. I'm sure there are still examples of it in online tutorials, but it has been several years since this was considered normal practice.